test/suite/test_chunkcache01.py

```diff
@@ -58,11 +57,11 @@ def get_stat(self, stat):
     def conn_config(self):
         if not os.path.exists('bucket1'):
             os.mkdir('bucket1')
-
```

Comment:
Nitpick, please leave this blank line

Reply:
Updated in [86a8f5c](https://github.com/wiredtiger/wiredtiger/pull/9597/commits/86a8f5c1d987125f322014d7f8b9a495147397f5)